scss/_variables.scss

```diff
@@ -286,6 +286,18 @@ $grid-hovered-bg: $table-bg-hover !default;
 $grid-hovered-border: $grid-border !default;
 
 
+// Progress
+$progress-text: $body-color !default;
+$progress-bg: $progress-bg !default;
+$progress-border: transparent !default;
+$progress-gradient: none !default;
+
+$progress-selected-text: $progress-bar-color !default;
+$progress-selected-bg:  $progress-bar-bg !default;
+$progress-selected-border: transparent !default;
+$progress-selected-gradient: none !default;
```

Comment:
How about naming these `$progress-fill-*` instead of `$progress-selected-*`?
Also, do we need the border / gradient vars, they don't seem to be used anywhere yet?